service/controller/resource/monitoring/scrapeconfigs/resource.go

```diff
@@ -296,6 +297,10 @@ func listTargetsToIgnore(ctx context.Context, ctrlClient client.Client, cluster
 		if err != nil {
 			return nil, microerror.Mask(err)
 		}
+		bundleWithKubeProxyExporterVersion, err := semver.Parse("0.8.3")
```

Comment:
Note that this will be released once https://github.com/giantswarm/observability-bundle/pull/113 is merged